lib/index.js

```diff
@@ -8,6 +8,7 @@ var foldl = require('@ndhoule/foldl');
 var is = require('is');
 var integration = require('@segment/analytics.js-integration');
 var omit = require('omit');
+var newDate = require('new-date');
```

Comment:
We would like to avoid adding dependencies whenever possible. Is there a reason we cant use the native `Date` constructor?

Reply:
I'm using this to make `track()` work as similar as `trait()`. In `trait()`, we have a `createdAt` var that gets its value from `identify.created()`. Since the `track` object doesn't have a `created()` (at least that why I know from logging the object and searching the docs), we need to parse the date ourselves from the properties. I don't have the code right now, but if I'm not wrong, I remember `created()` used `newDate` to convert date, which I like since `newDate` allows extra input types. :)

Reply:
@diegoserranoa The track object should have a `createdAt` timestamp. Does that seem like it will work for you?